hoot-core/src/main/cpp/hoot/core/cmd/ConvertCmd.cpp

```diff
@@ -127,6 +132,50 @@ class ConvertCmd : public BaseCommand
     return 0;
   }
 
+  boost::shared_ptr<ElementCriterion> _getStreamingCriterion(const QStringList ops)
+  {
+    QStringList criterionNames;
+    Factory& factory = Factory::getInstance();
+    for (int i = 0; i < ops.size(); i++)
+    {
+      const QString opName = ops[i];
+      LOG_VART(opName);
+      if (!opName.trimmed().isEmpty())
+      {
+        if (factory.hasBase<ElementCriterion>(opName.toStdString()))
+        {
+          criterionNames.append(opName);
+        }
```

Comment:
per recent discussion, will merge as is and let Jason's upcoming changes solve the issue.